.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+**Please list the related package(s)**
+e.g. argo-checkout, argo-checkout-react
+
+**If this is related to specific APIs or components, please list them here**
```

Comment:
I like the way Polaris's issue template does this a bit better, with proper headings and HTML comments for the body so they never show up in the rendered issue: https://raw.githubusercontent.com/Shopify/polaris-react/main/.github/ISSUE_TEMPLATE/ISSUE.md

Reply:
Good point, will update the samples later today. Thanks for sharing the Polaris examples!